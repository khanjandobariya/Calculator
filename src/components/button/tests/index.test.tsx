import Button from '../index';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Button', () => {
  test('should match to button snapshot', () => {
    const component = renderer
      .create(<Button label="Demo" onPress={() => {}} />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
