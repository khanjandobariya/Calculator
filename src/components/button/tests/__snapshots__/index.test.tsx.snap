// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match to button snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#9B9B9B",
      "borderRadius": 500,
      "height": 128.57142857142856,
      "justifyContent": "center",
      "opacity": 1,
      "width": 128.57142857142856,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontFamily": "Gilroy-Medium",
          "fontSize": 47.14285714285714,
        },
        undefined,
        undefined,
      ]
    }
  >
    Demo
  </Text>
</View>
`;
